### ISSUE TITLE
Too much text is quoted

### ISSUE THREAD

**Issue by mikemccand** [apache/lucene-jira-archive#93]
See [this comment](https://github.com/mocobeta/forks-migration-test/issues/2321#issuecomment-1197365531), where it looks like all text was quoted (prefixed with `> ` in GitHub markdown).

But if you look at the original Jira comment, there are separating newlines between the quoted parts and the un-quoted replies.  Yet the conversion quoted too much?

**Comment by mikemccand** [comment]
OK so this is a case of using the wrong quote character for Jira (`>`), and yet on migrating to GitHub issues, it is recognized as a quote, but then it quotes  more than intended (even text after the newline of the line that had the `> ` prefix).

**Comment by mikemccand** [comment]
We might be able to work around this by inserting an extra newline when we detect `^> ...` at the start of a line?

So from the above issue, we would map:

```
> EG running merges (or any still-open files) should not be sync'd.
Files that are still being written should not be synced, that's kinda obvious.
```

to:

```
> EG running merges (or any still-open files) should not be sync'd.

Files that are still being written should not be synced, that's kinda obvious.
```

I can confirm it works here, i.e. (wrong rendering):

> EG running merges (or any still-open files) should not be sync'd.
Files that are still being written should not be synced, that's kinda obvious.

vs. right rendering:

> EG running merges (or any still-open files) should not be sync'd.

Files that are still being written should not be synced, that's kinda obvious.

**Comment by mocobeta** [comment]
Incorrect quotes are misleading. I'd like to manage to address this...

I think it is difficult to correctly insert line feeds like this.

> EG running merges (or any still-open files) should not be sync'd.

Files that are still being written should not be synced, that's kinda obvious.

--

Meanwhile, escaping `>` with `&gt;` would be easy and safe I think.

&gt; EG running merges (or any still-open files) should not be sync'd.
Files that are still being written should not be synced, that's kinda obvious.

I already did it for [XML/HTML tag-like strings](https://github.com/apache/lucene-jira-archive/pull/32/files), and the tweak works fine; will try to apply the same fix for this.

**Comment by mocobeta** [comment]
I opened workaround PR #146.

**Comment by mocobeta** [comment]
Merged #146 - hope you don't mind.

**Comment by mikemccand** [comment]
Thanks @mocobeta!  Of course I don't mind -- progress not perfection ;)

**Comment by mocobeta** [comment]
I think this is fixed in #146.
